pkg/skaffold/util/once.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"sync"
+)
+
+type Once struct {
```

Comment:
Doc comment.  I really don't like this name as it is confusing given sync.Once.

Reply:
I read through the Go style guide. Package names are supposed to be considered part of an exported type. So here `util.Once` is identifiable as different from `sync.Once`. Also the point about surrounding context help disambiguate types instead of packing information in the type names. I made the change since you insisted on it 🙂